webpack.config.js

```diff
@@ -16,76 +17,58 @@ const env = config.parsed,
-    'ocp-ams-image-g111': './src/entrypoints/entry-ocp-ams-image-g111.js',
-    'oculus-oc5-signage': './src/entrypoints/entry-oculus-oc5-signage.js',
-    'oculus-oc5-registration': './src/entrypoints/entry-oculus-oc5-registration.js',
-    'oc6-oculus-signage': './src/entrypoints/entry-oc6-oculus-signage.js',
-    'oc6-oculus-signage-220b': './src/entrypoints/entry-oc6-oculus-signage-220b.js',
-    'oc6-oculus-signage-220c': './src/entrypoints/entry-oc6-oculus-signage-220c.js',
-    'oc6-oculus-signage-210f': './src/entrypoints/entry-oc6-oculus-signage-210f.js',
-    'oc6-oculus-signage-210h': './src/entrypoints/entry-oc6-oculus-signage-210h.js',
-    'oc6-oculus-signage-tt': './src/entrypoints/entry-oc6-oculus-signage-tt.js',
-    'oc6-oculus-signage-sp': './src/entrypoints/entry-oc6-oculus-signage-sp.js',
-    'oculus-oc6-sessions': './src/entrypoints/entry-oculus-oc6-sessions.js',
-    'oculus-oc6-schedule': './src/entrypoints/entry-oculus-oc6-schedule.js',
-    'oc6-oculus-image-eod1': './src/entrypoints/entry-oc6-oculus-image-eod1.js',
-    'oc6-oculus-image-eod2': './src/entrypoints/entry-oc6-oculus-image-eod2.js',
-    'oc6-oculus-image-sessionfull': './src/entrypoints/entry-oc6-oculus-image-sessionfull.js',
-    'f8-schedule': './src/entrypoints/entry-f8-schedule.js',
-    'f8-sessions': './src/entrypoints/entry-f8-sessions.js',*/
+const getHTMLFiles = (dir, fileList = []) => {
+    const files = fs.readdirSync(dir);
+    files.forEach((file) => {
+        const filePath = path.join(dir, file);
+        const isDirectory = fs.statSync(filePath).isDirectory();
+
+        if (isDirectory) {
+            getHTMLFiles(filePath, fileList);
+        } else {
+            if (path.extname(filePath).toLowerCase() === '.html') {
+                const relativePath = path.relative('templates', filePath);
+                fileList.push(relativePath);
+            }
+        }
+
+    });
+    return fileList;
+};
+
+const htmlTemplatesFiles = getHTMLFiles('./templates');
+
+const generateTemplateEntryPoints = (templateList) => {
+
+    const defaultEntryPoints = {
+        'config-admin': './src/config-admin.js',
+        'image-landscape': './src/entrypoints/entry-image-landscape.js',
+        'image': './src/entrypoints/entry-image.js',
+        'banner': './src/entrypoints/entry-banner.js',
+        'schedule': './src/entrypoints/entry-schedule.js',
+    };
+
+    const entryPoints = {};
+
+    templateList.forEach((templatePath) => {
+        const summitIdRegex = /\/(\d+)\//;
+        const entryNameRegex = /\/([\w-]+)\.html$/;
+        const summitIdMatch = templatePath.match(summitIdRegex);
+        const entryNameMatch = templatePath.match(entryNameRegex);
+        // if the template is in a folder that's not only digits, do not add it
+        if (summitIdMatch && entryNameMatch) {
+            const entryName = entryNameMatch[1];
+            entryPoints[entryName] = `./src/entrypoints/entry-${entryName}`;
```

Comment:
you need to check that entry point file name exits before add it, if doesnt exists discard it